Libraries/Animated/src/nodes/AnimatedTransform.js

```diff
@@ -87,6 +87,22 @@ class AnimatedTransform extends AnimatedWithChildren {
     super.__detach();
   }
 
+  __transformDataType(value: any): number {
```

Comment:
Could you extract transformDataType to NativeAnimatedHelpers and use it here https://github.com/facebook/react-native/blob/4148976a83ab96029de1b26b515ff95d3cb58c10/Libraries/Animated/src/nodes/AnimatedInterpolation.js#L356 also.

Reply:
OK I extracted to the common file. Interpolation works slightly differently (on an array) so it still needed the mapping function. This is from so long ago that I wasn't able to get the tests running again so maybe if you can check that it works.

Reply:
Yea sounds good, the main thing that is nice to extract out to be able to re-use is the deg to number conversion part. 👍